puzzle-utils/words/dropquote.py

```diff
@@ -169,40 +179,31 @@ def snapshot(self):
                 word.possibilities))
         
         return snapshot
+    
+    def print(self):
+        print('========== LETTERS ==========')
+        longest_column = 0
+        for column in self.letters:
+            if longest_column < len(column):
+                longest_column = len(column)
```

Comment:
You should be able to just use [max()](https://docs.python.org/3/library/functions.html#max) for this